### ISSUE TITLE
README Changes

### ISSUE THREAD

**Issue by srbhr** [Git-Commit-Show/100-Days-of-Code#232]
We're starting a new iteration of 100 Days of Code and we need to change the readme for this. 
Join in the discord for more information on it: dsc.gg/invide

**Comment by Bhavya0304** [comment]
can I contribute to this issue?

**Comment by srbhr** [comment]
Yes, everyone can contribute. But there are important changes that needs to be discussed. Please make sure it's not just a way to get away with Hacktoberfest swags. We at Invide, do not condone this behavior.

**Comment by srbhr** [comment]
Okay @Bhavya0304 please do suggest the changes and what all you'd like to work upon. Also meanwhile join our discord server: dsc.gg/invide
